src/components/TagsManager.tsx

```diff
@@ -2,21 +2,23 @@ import { Toaster } from "@/components/ui/toaster.tsx";
 import ContentWrapper from "@/components/ContentWrapper.tsx";
 import { TagsContent } from "@/components/TagsContent.tsx";
 import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs.tsx";
-import { useState } from "react";
 import PrimaryActionButton from "@/components/buttons/PrimaryActionButton.tsx";
 import { PlusCircle } from "lucide-react";
 import CreateOrModifyTagDialog from "@/components/CreateOrModifyTagDialog.tsx";
+import {useModalStore} from '@/store/modalStore.ts';
```

Comment:
also add eslint rule for better object forrmating

Reply:
There is another task (117) for the eslint rules.